apps/trading/pages/_app.page.tsx

```diff
@@ -28,7 +28,7 @@ function AppBody({ Component, pageProps }: AppProps) {
 
   return (
     <ThemeContext.Provider value={theme}>
-      <div className="h-full dark:bg-black dark:text-white-60 bg-white relative z-0 text-black-60 grid grid-rows-[min-content,1fr]">
+      <div className="h-full bg-black dark:text-white-60 bg-white relative z-0 text-black-60 grid grid-rows-[min-content,1fr]">
```

Comment:
Chore (blocking): this now has both classes `bg-black` and `bg-white`. Either one of them needs removing entirely, or one of them is needs a `dark:` prefix. You might also need to look at text colour.

Reply:
Thanks, have resolved that now.